ceph.spec.in

```diff
@@ -804,7 +804,6 @@ rm -rf $RPM_BUILD_ROOT
 
 
 %post
```

Comment:
The "ceph" RPM should be 100% empty, and the "ceph-base" package should contain ceph.target.

You're right, we should add the systemd scriptlets to the ceph-osd/ceph-mon/ceph-mds packages